layouts/shortcodes/carousel.html

```diff
@@ -0,0 +1,38 @@
+<div id="carouselExampleCaptions" class="carousel slide relative" data-bs-ride="carousel">
```

Comment:
Here, the ID should either be dynamically generated or user provided. If we leave it like this, we cannot have multiple carousels in the same page, or the controls from either one will just act on the first.

Reply:
nice one